changelogs/fragments/20__mm-feature__add-provision-virtual-esxi.yml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - provision_virtual_esxi - Added new role for provisioning a VM and installing ESXi on it. Added integration tests for role
```

Comment:
I think there is no need to mention about the test in the description. It's for users to know what's new in the collection and I guess users won't much care about test change.